src/views/LobbyView.vue

```diff
@@ -55,7 +61,7 @@ import { mapGetters, mapState } from 'vuex';
 import LobbyPlayerIndicator from '@/components/LobbyPlayerIndicator.vue';
 
 export default {
-  name: 'Lobby',
+  name: 'LobbyView',
```

Comment:
Aligned all component names with the component naming guidelines. 2 words, and for Cuttle specifically we can use `${name}View` for single word components, and to denote a top level view component.